AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -22,53 +25,76 @@ extern BOOL CGPointIsNull(CGPoint point)
   return isnan(point.x) && isnan(point.y);
 }
 
+@interface ASLayout ()
+
+/**
+ * A boolean describing if the current layout has been flattened.
+ */
+@property (nonatomic, getter=isFlattened) BOOL flattened;
```

Comment:
Yeah, it's best we keep it private for now. If we decide to keep around the initial layout tree that contains both layout spec and node layouts, I could see this being useful. However, in the filtered node layout tree result, there really isn't a need to expose this property.